lib/src/Base/Func/NormInfEnumerateFunction.cxx

```diff
@@ -143,6 +144,16 @@ UnsignedInteger NormInfEnumerateFunction::getMaximumDegreeStrataIndex(const Unsi
   return maximumDegree / getDimension();
 }
 
+/* The marginal enumerate function */
+EnumerateFunction NormInfEnumerateFunction::getMarginal(const Indices & indices) const
+{
+  const UnsignedInteger inputDimension = getDimension();
+  indices.check(inputDimension);
```

Comment:
and here

Reply:
Very true: fixed at a3e4351f503ea9976732960cfb0f4917c6b05d94.